pkg/cloud/azure/actuators/machine/reconciler_test.go

```diff
@@ -706,6 +706,8 @@ func TestMachineUpdateWithProvisionsngFailedNic(t *testing.T) {
 			mockCtrl := gomock.NewController(t)
 			networkSvc := mock_azure.NewMockService(mockCtrl)
 
+			testCtx := context.TODO()
```

Comment:
Nit, we will never replace a test `context.TODO()`, would it make sense to just use `context.Background()` instead?